cli/cmd/auth.go

```diff
@@ -80,9 +82,9 @@ Example:
 func init() {
 	rootCmd.AddCommand(authCmd)
 
-	endPoint = authCmd.Flags().StringP("endpoint", "e", "", "The endpoint exposed by keptn")
+	endPoint = authCmd.Flags().StringP("endpoint", "e", "", "The endpoint exposed by the Keptn installation (e.g., api.keptn.127.0.0.1.xip.io)")
 	authCmd.MarkFlagRequired("endpoint")
-	apiToken = authCmd.Flags().StringP("api-token", "a", "", "The API token provided by keptn")
+	apiToken = authCmd.Flags().StringP("api-token", "a", "", "The API token to communicate with the by Keptn installation")
```

Comment:
typo: "...with the by Keptn installation"

Reply:
fixed